src/composition.rs

```diff
@@ -612,20 +612,18 @@ impl ToString for Composition {
 }
 
 impl Composition {
-    #[allow(dead_code)]
-    fn new(file: ast::File) -> Self {
+    pub(crate) fn new(file: ast::File) -> Self {
         Self { file }
     }
 
     /// Initialize an ast::File for use in composition.
     ///
     /// This must be called before any other composition can be made, as it'll set up the
     /// statement that will be managed by composition.
-    #[allow(dead_code)]
-    fn initialize(
+    pub(crate) fn initialize(
         &mut self,
-        bucket: &str,
-        measurement: Option<&str>,
```

Comment:
Since we receive the command_params as Strings, it didn't make sense to convert to &str...when we just convert it back to a String again.